src/backend/gl/src/window/surfman.rs

```diff
@@ -44,12 +42,13 @@ impl fmt::Debug for Instance {
 }
 
 impl Instance {
-    fn get_default_context_attributes() -> sm::ContextAttributes {
+    fn get_default_context_attributes() -> sm::ContextAttributes<'static> {
         sm::ContextAttributes {
             version: sm::GLVersion::new(3, 3), // TODO: Figure out how to determine GL version
             // TODO: Skipping COMPATIBILITY_PROFILE for now, because it panics with a TODO, but
             // that is probably something we want to provide later.
             flags: sm::ContextAttributeFlags::ALPHA,
+            share_with: None,
```

Comment:
please switch to the upstream surfman, it has a better API from what I made in my fork initially

Reply:
:+1: Sounds good.